resources/js/components/CustomProperties.vue

```diff
@@ -9,63 +9,63 @@
 </template>
 
 <script>
-import CustomPropertiesModal from './CustomPropertiesModal'
-import tap from 'lodash/tap'
-import get from 'lodash/get'
-import set from 'lodash/set'
+  import CustomPropertiesModal from './CustomPropertiesModal'
+  import tap from 'lodash/tap'
+  import get from 'lodash/get'
+  import set from 'lodash/set'
```

Comment:
@stepanenko3 still has some spacing issues here.

Reply:
@bkintanar I don't see a problem right now

Perhaps this refers to the indentation inside the script tag, but it should be, because the script is nested in the tag

Please clarify

Reply:
You are correct @stepanenko3. I apologize. I'll continue to review the PR.